spring-cloud-sleuth-core/src/test/java/org/springframework/cloud/sleuth/instrument/web/client/integration/WebClientTests.java

```diff
@@ -671,13 +670,13 @@ public void clear() {
 	@RestController
 	public static class WebClientController {
 
-		@RequestMapping(value = "/issue1462", method = RequestMethod.GET)
+		@GetMapping("/issue1462")
 		public ResponseEntity<String> issue1462() {
 			System.out.println("GOT IT");
 			return ResponseEntity.status(499).body("issue1462");
 		}
 
-		@RequestMapping(value = { "/skip", "/doNotSkip" }, method = RequestMethod.GET)
+		@GetMapping({ "/skip", "/doNotSkip" })
```

Comment:
In this context, it's OK to remove the extra `value = `, because Rewrite knows that `value` is a special word in Java that can be omitted if the annotation arguments have been reduced to one.